proton-c/bindings/python/proton/__init__.py

```diff
@@ -2581,6 +2581,9 @@ def close(self):
     """
     self._update_cond()
     pn_connection_close(self._impl)
+    if hasattr(self, '_session_policy'):
```

Comment:
The issue with just using 'self._session_policy = None' is that it would create a useless attribute at his point if it doesn't already exist.

But either solution works.